style.css

```diff
@@ -515,3 +519,326 @@ footer {
+  }
+
+  .about-me {
+    height: 1522px;
+  }
+
+  .contact-me {
+    height: 879px;
+  }
+
+  .works {
+    height: 3283px;
+  }
+
+  .main-heading {
+    height: 653.32px;
+  }
+
+  .social {
+    width: 0;
+  }
+
+  .action {
+    margin: 0;
+  }
+}
+
+@media (max-width: 767px) {
+  .nav-list,
+  .form-img-center,
+  .form-desktop-image,
+  .desktop-img,
+  .aboutme2-desktop-img {
+    display: none;
+  }
+
+  .intro {
+    background-image: url('./images/header-illsutration-mobile.svg');
+    background-repeat: no-repeat;
+    background-size: contain;
+  }
+
+  .contact-heading {
+    margin-top: 0;
+  }
+}
+
+@media (min-width: 768px) {
+  .header {
+    height: 72px;
+    width: 100%;
+  }
+
+  .nav {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .nav-list {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 255px;
+    height: 40px;
+    list-style: none;
+    gap: 8px;
+  }
+
+  .nav-list--item {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 63px;
+    height: 20px;
+  }
+
+  .nav-list--item a {
+    padding: 10px 12px;
+    text-decoration: none;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+  }
+
+  .menu-icon {
+    display: none;
+  }
+
+  .main-heading {
+    background-image: url('./images/Header-llustration-desktop.svg');
+    background-position: top;
+    background-size: 100%;
+    background-repeat: no-repeat;
+    height: 1059px;
+  }
+
+  .intro {
+    height: 800px;
+  }
+
+  .intro-container {
+    justify-content: start;
+    width: 65%;
+    height: 756px;
+    margin-top: 120px;
+  }
+
+  .intro-heading {
+    font-weight: 800;
+    font-size: 48px;
+    line-height: 60px;
+  }
+
+  .intro-description p {
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    padding: 0 50px;
+  }
+
+  .social {
+    width: 200px;
+    height: 40px;
+  }
+
+  .social-links {
+    margin-bottom: 0;
+  }
+
+  .social-links :nth-child(1) {
+    order: 4;
+  }
+
+  .social-links :nth-child(2) {
+    order: 2;
+  }
+
+  .social-links :nth-child(3) {
+    order: 5;
+  }
+
+  .social-links :nth-child(4) {
+    order: 1;
+  }
+
+  .social-links :nth-child(5) {
+    order: 3;
+  }
+
+  .social-links a img {
+    width: 25px;
+    height: 25px;
+  }
+
+  a.action-button {
+    border: 1px solid #dfe1e6;
+    border-radius: 32px;
+    width: 40px;
+    height: 40px;
+  }
+
+  .action {
+    margin-top: 70px;
+  }
+
+  .main {
+    margin-top: 100px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 30px;
+    width: 100%;
+  }
+
+  .header-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 50%;
+  }
+
+  .header-container .indicator-container img {
+    display: none;
+  }
+
+  .works {
+    height: 100%;
+  }
+
+  .works-cards {
+    grid-template-columns: auto auto auto;
+  }
```

Comment:
The cards seem to go off the screen on some desktop screens. As [required](https://github.com/microverseinc/curriculum-html-css/blob/933fcb5206004f4ecaf1a898c06fa616010cede4/portfolio/4_desktop_version.md?plain=1#L13), kindly adjust it to remain in place for desktop screen (768px or wider).

Your design:
![image](https://user-images.githubusercontent.com/20554583/228992911-cf459dfd-b384-4fa7-b5fa-cb906fb734e3.png)